packages/popover/src/Popover.tsx

```diff
@@ -1,7 +1,15 @@
-import React, { ReactElement } from 'react';
-import Popper, { IContent } from '@xcritical/popper';
+import React, {
+  useState,
+  useCallback,
+  useEffect,
+  useRef,
+} from 'react';
 
-import { IPopover } from './interfaces';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why is this comment written here?

Reply:
@zima-snow Removed this dependency, thanks.